tutorials/basic-topics/kadane2D.md

```diff
@@ -24,14 +24,14 @@ For example, given the input:
 
 ```cpp
```

Comment:
To save your time. Like I mentioned before (point 3). What I'm expecting is a few LC problems as examples to go through the algo. Not a random example.

Reply:
Honestly I couldn't find any problems on 2D kadane in leetcode. I have followed the template now and made final changes. I feel that the article should be enough and would give enough idea on this algorithm.  This article can be improved in future. I would be eagerly waiting to improve this one. Please accept my effort, I have tried explaining each step clearly